.githooks/pre-commit

```diff
@@ -14,56 +14,60 @@ if [ -z "$STAGED_FILES" ]; then
   exit 0
 fi
 
-# Find the files not declared in `$FILE_IGNORED_PAGES`
-for translated in $(echo "$STAGED_FILES" | grep "$SRC_PATTERN"); do
-  if ! grep --quiet "$translated" "$FILE_IGNORED_PAGES"; then
-    FILES_NOT_IGNORED+=("$translated")
-  fi
-done
-
-if [ ${#FILES_NOT_IGNORED[@]} -ne 0 ]; then
-  # Warn user about files missing from `$FILE_IGNORED_PAGES`
+FILES_TRANSLATED=$(echo "$STAGED_FILES" | grep "$SRC_PATTERN")
+readonly FILES_TRANSLATED
 
-  echo
-  echo "******* WARNING *******"
-  echo
-  echo "You have translated these files:"
-  echo
+if [ -n "$FILES_TRANSLATED" ]; then
```

Comment:
Is there a reason why you didn't use the same approach here as the block:
```
if [ -z "$STAGED_FILES" ]; then
  exit 0
fi
```

? Exit right now if none of the `$STAGED_FILES` are included in `guides/`? It looks more readable to me than the embedded `if`

Reply:
👆 The question is still valid but this has been opened for a while and it's better than what we currently have, so I'll merge.